doc/reference/std/actor.md

```diff
@@ -861,6 +947,506 @@ The default path for the ensemble admin public key; defaults to `$GERBIL_PATH/en
+ actor-server: (srv (current-actor-server)))
+```
+
+Restart a supervisor and the parts of the ensemble supervised by it.
+
+### ensemble-supervisor-update-server-config!
+```
+(ensemble-supervisor-update-server-config!
+ supervisor: (super (ensemble-domain-supervisor))
+ server: server-id
+ config: cfg
+ mode: (mode 'upsert)
+ restart: (restart? #t)
+ actor-server: (srv (current-actor-server)))
+```
+
+Update a server configuration for a supervisor
+
+### ensemble-supervisor-get-server-config
+```
+(ensemble-supervisor-get-server-config
+ supervisor: (super (ensemble-domain-supervisor))
+ server: server-id
+ actor-server: (srv (current-actor-server)))
+```
+
+Retrieve a server configuration from a supervisor.
+
+### ensemble-update-config!
+```
+(ensemble-update-config!
+ config: config
+ mode: (mode 'upsert)
+ actor-server: (srv (current-actor-server)))
+```
+
+Update the ensemble configuration for all known supervisors.
+
+### ensemble-supervisor-update-config!
+```
+(ensemble-supervisor-update-config!
+ supervisor: (super (ensemble-domain-supervisor))
+ config: config
+ mode: (mode 'upsert)
+ actor-server: (srv (current-actor-server)))
+```
+
+Update the ensemble configuration for a specific supervisor.
+
+### ensemble-get-config
+```
+(ensemble-get-config
+ actor-server: (srv (current-actor-server)))
+```
+
+Collect the ensemble configuration from all known supervisors.
+
+### ensemble-supervisor-get-config
+```
+(ensemble-supervisor-get-config
+ supervisor: (super (ensemble-domain-supervisor))
+ actor-server: (srv (current-actor-server)))
+```
+
+Retrieve the ensemble configuration from a specific supervisor.
+
+### ensemble-upload-executable!
+```
+(ensemble-upload-executable!
+ path: executable-gz-path
+ deployment-path: deployment-path
+ actor-server: (srv (current-actor-server)))
+```
+
+Upload an executable to all ensemble supervisors.
+
+### ensemble-supervisor-upload-executable!
+```
+(ensemble-supervisor-upload-executable!
+ supervisor: (super (ensemble-domain-supervisor))
+ path: executable-gz-path
+ deployment-path: deployment-path
+ actor-server: (srv (current-actor-server)))
+```
+Upload an executable to a specific superisor.
+
+### ensemble-upload-environment!
+```
+(ensemble-upload-environment!
+ path: env-targz-path
+ deployment-path: deployment-path
+ actor-server: (srv (current-actor-server)))
+```
+Upload a gerbil environment to all known supervisor.
+
+### ensemble-supervisor-upload-environment!
+```
+(ensemble-supervisor-upload-environment!
+ supervisor: (super (ensemble-domain-supervisor))
+ path: env-targz-path
+ deployment-path: deployment-path
+ actor-server: (srv (current-actor-server)))
+```
+
+Upload a gerbil environment to a specific supervisor.
+
+### ensemble-upload-filesystem-overlay!
+```
+(ensemble-upload-filesystem-overlay!
+ fs: fs-targz-path
+ path: deployment-path
+ actor-server: (srv (current-actor-server)))
+```
+
+Upload a new server accessible filesystem overlay to all known supervisors.
```

Comment:
Is that documented elsewhere?

Reply:
yes, in the advanced ensemble tutorial by example.